cmd/tailscale/cli/remove.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"tailscale.com/cmd/tailscale/cli/ffcomplete"
+	"tailscale.com/ipn"
+)
+
+var removeCmd = &ffcli.Command{
+	Name:       "remove",
+	ShortUsage: "tailscale remove <id>",
```

Comment:
Can we put this "remove" as a subcommand under something else? I'd rather it not be so prominent in the top-level docs.